### PR TITLE
Components: update dragging events

### PR DESCRIPTION
**Pull request by sergepilipchuk**

### CODE REVIEW

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md [hunk 1]
```diff
@@ -22,25 +22,25 @@ Indicates if the dragged row is dropped inside another row. Available only for t
...
 ##### field(e.fromData): any
-Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the source UI component.
+Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the source UI component. Works only for dragging between components.
 
 ##### field(e.fromIndex): Number
 The row's original position. If the row is dragged from another UI component, the value is **-1**.
 
 ##### field(e.itemData): any
-The dragged row's data.
+The row's data before dragging.
```
Comment (williamvinogradov) [inline api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md]: Just note it's not a comment needed to fix.

In the Grid and TreeList, the itemData before drag-n-drop equals the "itemData" after drag-n-drop.
Because in these components, the "itemData" doesn't contain any fields that will change during drag-n-drop interaction.

But I agree that mentioning that the "itemData" is data **before** drag-n-drop's interaction will be better.

File: api-reference/10 UI Components/dxScheduler/1 Configuration/appointmentDragging/onDragEnd.md [hunk 2]
```diff
@@ -19,25 +19,25 @@ The UI component's instance.
...
-The instance of the appointment's target UI component.
+The instance of the appointment's target UI component. Works only for dragging between components.
 
 ##### field(e.toData): any
-Custom [data](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/appointmentDragging/data.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/appointmentDragging/#data') associated with the target UI component.
+Custom [data](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/appointmentDragging/data.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/appointmentDragging/#data') associated with the target UI component. Works only for dragging between components.
 
 ##### field(e.toItemData): any
-<!-- Description goes here -->
+The appointment's data after dragging.
```
Comment (williamvinogradov) [inline api-reference/10 UI Components/dxScheduler/1 Configuration/appointmentDragging/onDragEnd.md:40]: In the "onDragEnd" handler, the user can cancel the drag-n-drop event by setting the ```e.cance = true;```

So, maybe it will be better to write here something like:
"The alleged appointment's data after dragging completes."
or
"The appointment's data after dragging completes".

Comment (sergepilipchuk) [reply api-reference/10 UI Components/dxScheduler/1 Configuration/appointmentDragging/onDragEnd.md:40]: I think the "The appointment's data after dragging completes" expression means the same as "The appointment's data after dragging."

**Review by williamvinogradov — APPROVED** [review]

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md [hunk 3]
```diff
@@ -22,25 +22,25 @@ Indicates if the dragged row is dropped inside another row. Available only for t
 #include common-ref-eventparam
 
 ##### field(e.fromComponent): dxSortable | dxDraggable
-The instance of the row's source UI component.
+The instance of the row's source UI component. Works only for dragging between components.
 
 ##### field(e.fromData): any
-Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the source UI component.
+Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the source UI component. Works only for dragging between components.
```
Comment (albertov05) [inline api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md]: We should avoid the -ing form if possible. Can you clarify the sentence so that it specifies what is being dragged between components?

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md (hunk 1 above)
Comment (albertov05) [inline api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md]: See my explanation above. Can we restore this line to the previous description?

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md [hunk 4]
```diff
@@ -22,25 +22,25 @@ Indicates if the dragged row is dropped inside another row. Available only for t
...
 ##### field(e.itemData): any
-The dragged row's data.
+The row's data before dragging.
 
 ##### field(e.itemElement): DxElement
 #include draganddrop-ref-itemelementparam
 
 ##### field(e.toComponent): dxSortable | dxDraggable
-The instance of the row's target UI component.
+The instance of the row's target UI component. Works only for dragging between components.
```
Comment (albertov05) [inline api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md]: See my comment on line 28 above.

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md [hunk 5]
```diff
@@ -22,25 +22,25 @@ Indicates if the dragged row is dropped inside another row. Available only for t
...
 
 ##### field(e.toData): any
-Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the target UI component.
+Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the target UI component. Works only for dragging between components.
```
Comment (albertov05) [inline api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md]: See my comment above.

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragEnd.md [hunk 6]
```diff
@@ -22,25 +22,25 @@ Indicates if the dragged row is dropped inside another row. Available only for t
 #include common-ref-eventparam
 
 ##### field(e.fromComponent): dxSortable | dxDraggable
-The instance of the row's source UI component.
+The instance of the row's source UI component. Works only for dragging between components.
```
Comment (albertov05) [inline api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragEnd.md]: See my comment above.

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragEnd.md [hunk 7]
```diff
@@ -22,25 +22,25 @@ Indicates if the dragged row is dropped inside another row. Available only for t
...
 
 ##### field(e.fromData): any
-Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the source UI component.
+Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the source UI component. Works only for dragging between components.
```
Comment (albertov05) [inline api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragEnd.md]: See my comment above.

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragEnd.md [hunk 8]
```diff
@@ -22,25 +22,25 @@ Indicates if the dragged row is dropped inside another row. Available only for t
...
 
 ##### field(e.fromIndex): Number
 The row's original position. If the row is dragged from another UI component, the value is **-1**.
 
 ##### field(e.itemData): any
-The data object of the row being dragged.
+The row's data before dragging.
```
Comment (albertov05) [inline api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragEnd.md]: See my comment above. In this case, if a row is dragged, does the data actually change? Please clarify and update your description.

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragEnd.md [hunk 9]
```diff
@@ -22,25 +22,25 @@ Indicates if the dragged row is dropped inside another row. Available only for t
...
 
 ##### field(e.itemElement): DxElement
 #include draganddrop-ref-itemelementparam
 
 ##### field(e.toComponent): dxSortable | dxDraggable
-The instance of the row's target UI component.
+The instance of the row's target UI component. Works only for dragging between components.
```
Comment (albertov05) [inline api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragEnd.md]: See my comment above.

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragEnd.md [hunk 10]
```diff
@@ -22,25 +22,25 @@ Indicates if the dragged row is dropped inside another row. Available only for t
...
 
 ##### field(e.toData): any
-Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the target UI component.
+Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the target UI component. Works only for dragging between components.
```
Comment (albertov05) [inline api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragEnd.md]: See my comment above.

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragMove.md [hunk 11]
```diff
@@ -22,25 +22,25 @@ Indicates if the dragged row is dropped inside another row. Available only for t
...
 ##### field(e.fromData): any
-Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the source UI component.
+Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the source UI component. Works only for dragging between components.
 
 ##### field(e.fromIndex): Number
 The row's original position. If the row is dragged from another UI component, the value is **-1**.
 
 ##### field(e.itemData): any
-The data object of the row being dragged.
+The row's data before dragging.
```
Comment (albertov05) [inline api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragMove.md]: See my comment on line 34 above.

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragMove.md [hunk 12]
```diff
@@ -22,25 +22,25 @@ Indicates if the dragged row is dropped inside another row. Available only for t
...
 
 ##### field(e.itemElement): DxElement
 #include draganddrop-ref-itemelementparam
 
 ##### field(e.toComponent): dxSortable | dxDraggable
-The instance of the row's target UI component.
+The instance of the row's target UI component. Works only for dragging between components.
```
Comment (albertov05) [inline api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragMove.md]: See my comment above.

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragMove.md [hunk 13]
```diff
@@ -22,25 +22,25 @@ Indicates if the dragged row is dropped inside another row. Available only for t
...
 
 ##### field(e.toData): any
-Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the target UI component.
+Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the target UI component. Works only for dragging between components.
```
Comment (albertov05) [inline api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragMove.md]: See my comment above.

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragStart.md [hunk 14]
```diff
@@ -19,13 +19,13 @@ The UI component's instance.
 #include common-ref-eventparam
 
 ##### field(e.fromData): any
-Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the source UI component.
+Custom [data](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/rowDragging/data.md '{basewidgetpath}/Configuration/rowDragging/#data') associated with the source UI component. Works only for dragging between components.
```
Comment (albertov05) [inline api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragStart.md]: See my comment above.

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragStart.md [hunk 15]
```diff
@@ -19,13 +19,13 @@ The UI component's instance.
...
 
 ##### field(e.fromIndex): Number
 The row's original position. If the row is dragged from another UI component, the value is **-1**.
 
 ##### field(e.itemData): any
-The data object of the row being dragged.
+The row's data before dragging.
```
Comment (albertov05) [inline api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragStart.md]: See my comment above.

File: api-reference/10 UI Components/dxDraggable/1 Configuration/onDragEnd.md [hunk 16]
```diff
@@ -23,13 +23,13 @@ The UI component that raised the event.
 #include common-ref-eventparam
```
Comment (albertov05) [inline api-reference/10 UI Components/dxDraggable/1 Configuration/onDragEnd.md:24]: Please update the following line above:

Line 8:
`A function that is called when drag gesture is finished.`

`A function that is called when a drag gesture is finished.`

File: api-reference/10 UI Components/dxDraggable/1 Configuration/onDragEnd.md [hunk 17]
```diff
@@ -23,13 +23,13 @@ The UI component that raised the event.
...
 
 ##### field(e.fromComponent): dxSortable | dxDraggable
-The instance of the item's source UI component.
+The instance of the item's source UI component. Works only for dragging between components.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxDraggable/1 Configuration/onDragEnd.md]: See my comment above.

File: api-reference/10 UI Components/dxDraggable/1 Configuration/onDragEnd.md [hunk 18]
```diff
@@ -23,13 +23,13 @@ The UI component that raised the event.
...
 
 ##### field(e.fromData): any
-Custom [data](/api-reference/10%20UI%20Components/DraggableBase/1%20Configuration/data.md '{basewidgetpath}/Configuration/#data') associated with the source UI component.
+Custom [data](/api-reference/10%20UI%20Components/DraggableBase/1%20Configuration/data.md '{basewidgetpath}/Configuration/#data') associated with the source UI component. Works only for dragging between components.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxDraggable/1 Configuration/onDragEnd.md]: See my comment above.

File: api-reference/10 UI Components/dxDraggable/1 Configuration/onDragEnd.md [hunk 19]
```diff
@@ -23,13 +23,13 @@ The UI component that raised the event.
...
 
 ##### field(e.itemData): any
-The dragged item's data. Note that you should specify the item data manually.
+The item's data before dragging. Note that you should specify the item data manually.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxDraggable/1 Configuration/onDragEnd.md]: See my previous comments regarding the -ing form. Can we restore the previous text?

File: api-reference/10 UI Components/dxDraggable/1 Configuration/onDragEnd.md [hunk 20]
```diff
@@ -38,10 +38,10 @@ The dragged item's data. Note that you should specify the item data manually.
 Model data. Available only if you use Knockout.
 
 ##### field(e.toComponent): dxSortable | dxDraggable
-The instance of the item's target UI component.
+The instance of the item's target UI component. Works only for dragging between components.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxDraggable/1 Configuration/onDragEnd.md]: See my comment above.

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md (hunk 3 above)
Comment (sergepilipchuk) [reply api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md]: To avoid -ing, I can rewrite the sentence in the following way: "Works only for drag and drop elements between components. In the following demo you can see and try how drag rows between 2 components: https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/DnDBetweenGrids/

Comment (sergepilipchuk) [reply api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md]: To avoid -ing, I can rewrite the sentence in the following way: "Works only for drag and drop elements between components. In the following demo you can see and try how drag rows between 2 components: https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/DnDBetweenGrids/

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md (hunk 1 above)
Comment (sergepilipchuk) [reply api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md]: No! The key words in this sentence are "before dragging". We can rewrite it in the following way: The row's data before a drag.

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md (hunk 3 above)
Comment (albertov05) [reply api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md]: I would rephrase it as:  
Works only for element drag and drop between components. In the following demo you can see how to drag rows between two components: https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/DnDBetweenGrids/.

If you're okay with this, can you replace all instances of this throughout the PR?

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md (hunk 1 above)
Comment (albertov05) [reply api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md]: I see. How about:
`The row's data before a drag operation.`

(Or "a drag action", if you think that phrase is better.)

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md (hunk 3 above)
Comment (sergepilipchuk) [reply api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md]: I agree with the first sentence: Works only for element drag and drop between components.
Second part was only to explain how it works)

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md (hunk 1 above)
Comment (sergepilipchuk) [reply api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md]: Yes, thanks)

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md (hunk 3 above)
Comment (albertov05) [reply api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md]: First sentence: Let's go with my suggestion then!
Second sentence: I primarily wanted to update the "see and try" (we normally don't use this combination in American English usage). How about:
The following demo shows how to drag rows between two components: https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/DnDBetweenGrids/.

Comment (sergepilipchuk) [reply api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md]: I specified second sentence about demo only for you) In the table of options I cannot specify this sentence.

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md (hunk 1 above)
Comment (albertov05) [reply api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md]: Okay, please use "find and replace to replace all instances of this phrase)

File: api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md (hunk 3 above)
Comment (albertov05) [reply api-reference/10 UI Components/GridBase/1 Configuration/rowDragging/onDragChange.md]: Oh okay, I get it. Thanks.

File: api-reference/10 UI Components/dxDraggable/1 Configuration/onDragMove.md [hunk 21]
```diff
@@ -23,13 +23,13 @@ The UI component that raised the event.
...
-The instance of the item's source UI component.
+The instance of the item's source UI component. Works only for dragging between components.
 
 ##### field(e.fromData): any
-Custom [data](/api-reference/10%20UI%20Components/DraggableBase/1%20Configuration/data.md '{basewidgetpath}/Configuration/#data') associated with the source UI component.
+Custom [data](/api-reference/10%20UI%20Components/DraggableBase/1%20Configuration/data.md '{basewidgetpath}/Configuration/#data') associated with the source UI component. Works only for dragging between components.
 
 ##### field(e.itemData): any
-The dragged element's data. Note that you should specify the item data manually.
+The item's data before dragging. Note that you should specify the item data manually.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxDraggable/1 Configuration/onDragMove.md]:
```suggestion
The item's data before the drag operation. Note that you should specify the item data manually.
```

File: api-reference/10 UI Components/dxDraggable/1 Configuration/onDragStart.md [hunk 22]
```diff
@@ -23,10 +23,10 @@ The UI component that raised the event.
 #include common-ref-eventparam
 
 ##### field(e.fromData): any
-Custom [data](/api-reference/10%20UI%20Components/DraggableBase/1%20Configuration/data.md '{basewidgetpath}/Configuration/#data') associated with the source UI component.
+Custom [data](/api-reference/10%20UI%20Components/DraggableBase/1%20Configuration/data.md '{basewidgetpath}/Configuration/#data') associated with the source UI component. Works only for dragging between components.
 
 ##### field(e.itemData): any
-The dragged item's data. Note that you should specify the item data manually.
+The item's data before dragging. Note that you should specify the item data manually.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxDraggable/1 Configuration/onDragStart.md]:
```suggestion
The item's data before the drag operation. Note that you should specify the item data manually.
```

File: api-reference/10 UI Components/dxScheduler/1 Configuration/appointmentDragging/onDragEnd.md [hunk 23]
```diff
@@ -19,25 +19,25 @@ The UI component's instance.
...
-The instance of the appointment's source UI component.
+The instance of the appointment's source UI component. Works only for dragging between components.
 
 ##### field(e.fromData): any
-Custom [data](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/appointmentDragging/data.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/appointmentDragging/#data') associated with the source UI component.
+Custom [data](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/appointmentDragging/data.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/appointmentDragging/#data') associated with the source UI component. Works only for dragging between components.
 
 ##### field(e.itemData): any
-The dragged appointment's data.
+The appointment's data before dragging.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxScheduler/1 Configuration/appointmentDragging/onDragEnd.md]:
```suggestion
The appointment's data before the drag operation.
```

File: api-reference/10 UI Components/dxScheduler/1 Configuration/appointmentDragging/onDragEnd.md (hunk 2 above)
Comment (albertov05) [inline api-reference/10 UI Components/dxScheduler/1 Configuration/appointmentDragging/onDragEnd.md:40]:
```suggestion
The appointment's data after the drag operation.
```

File: api-reference/10 UI Components/dxScheduler/1 Configuration/appointmentDragging/onDragStart.md [hunk 24]
```diff
@@ -19,10 +19,10 @@ The UI component's instance.
 #include common-ref-eventparam
 
 ##### field(e.fromData): any
-Custom [data](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/appointmentDragging/data.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/appointmentDragging/#data') associated with the source UI component.
+Custom [data](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/appointmentDragging/data.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/appointmentDragging/#data') associated with the source UI component. Works only for dragging between components.
 
 ##### field(e.itemData): any
-The data object of the appointment being dragged.
+The appointment's data before dragging.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxScheduler/1 Configuration/appointmentDragging/onDragStart.md]:
```suggestion
The appointment's data before the drag operation.
```

File: api-reference/10 UI Components/dxSortable/1 Configuration/onDragChange.md [hunk 25]
```diff
@@ -26,16 +26,16 @@ Indicates if the dragged item is dropped inside another item.
...
 ##### field(e.fromData): any
-Custom [data](/api-reference/10%20UI%20Components/DraggableBase/1%20Configuration/data.md '{basewidgetpath}/Configuration/#data') associated with the source UI component.
+Custom [data](/api-reference/10%20UI%20Components/DraggableBase/1%20Configuration/data.md '{basewidgetpath}/Configuration/#data') associated with the source UI component. Works only for dragging between components.
 
 ##### field(e.fromIndex): Number
 The item's original position. If the item is dragged from another UI component, the value is **-1**.
 
 ##### field(e.itemData): any
-The dragged item's data. Note that you should specify the item data manually.
+The item's data before dragging. Note that you should specify the item data manually.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxSortable/1 Configuration/onDragChange.md]:
```suggestion
The item's data before the drag operation. Note that you should specify the item data manually.
```

File: api-reference/10 UI Components/dxSortable/1 Configuration/onDragEnd.md [hunk 26]
```diff
@@ -26,16 +26,16 @@ Indicates if the dragged item is dropped inside another item.
...
 ##### field(e.fromData): any
-Custom [data](/api-reference/10%20UI%20Components/DraggableBase/1%20Configuration/data.md '{basewidgetpath}/Configuration/#data') associated with the source UI component.
+Custom [data](/api-reference/10%20UI%20Components/DraggableBase/1%20Configuration/data.md '{basewidgetpath}/Configuration/#data') associated with the source UI component. Works only for dragging between components.
 
 ##### field(e.fromIndex): Number
 The item's original position. If the item is dragged from another UI component, the value is **-1**.
 
 ##### field(e.itemData): any
-The dragged item's data. Note that you should specify the item data manually.
+The item's data before dragging. Note that you should specify the item data manually.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxSortable/1 Configuration/onDragEnd.md]:
```suggestion
The item's data before the drag operation. Note that you should specify the item data manually.
```

File: api-reference/10 UI Components/dxSortable/1 Configuration/onDragMove.md [hunk 27]
```diff
@@ -26,16 +26,16 @@ Indicates if the dragged item is dropped inside another item.
...
 ##### field(e.fromData): any
-Custom [data](/api-reference/10%20UI%20Components/DraggableBase/1%20Configuration/data.md '{basewidgetpath}/Configuration/#data') associated with the source UI component.
+Custom [data](/api-reference/10%20UI%20Components/DraggableBase/1%20Configuration/data.md '{basewidgetpath}/Configuration/#data') associated with the source UI component. Works only for dragging between components.
 
 ##### field(e.fromIndex): Number
 The item's original position. If the item is dragged from another UI component, the value is **-1**.
 
 ##### field(e.itemData): any
-The dragged item's data. Note that you should specify the item data manually.
+The item's data before dragging. Note that you should specify the item data manually.
```
Comment (albertov05) [inline api-reference/10 UI Components/dxSortable/1 Configuration/onDragMove.md]:
```suggestion
The item's data before the drag operation. Note that you should specify the item data manually.
```

File: api-reference/10 UI Components/dxSortable/1 Configuration/onDragStart.md [hunk 28]
```diff
@@ -23,13 +23,13 @@ The UI component that raised the event.
 #include common-ref-eventparam
```
Comment (albertov05) [inline api-reference/10 UI Components/dxSortable/1 Configuration/onDragStart.md:24]: Please update the following line above:
`A function that is called when drag gesture is initialized.`

`A function that is called when a drag gesture is initialized.`
